cluster/aws/util.sh

```diff
@@ -1007,9 +1007,9 @@ function kube-up {
   # Wait 3 minutes for cluster to come up.  We hit it with a "highstate" after that to
   # make sure that everything is well configured.
   # TODO: Can we poll here?
-  echo "Waiting 3 minutes for cluster to settle"
+  echo "Waiting 1 minute for cluster to settle"
```

Comment:
(The risk/reward isn't worth it for removing it entirely; I think reducing it to 1 minute is a good stepping-stone)